framework/src/mysql.js

```diff
@@ -229,9 +229,17 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
 		}
 
 		if (params.sort) {
-			const [sortProp, sortOrder] = params.sort.split(':');
-			query.orderBy(sortProp, sortOrder);
-			query.whereNotNull(sortProp);
+			const sortList = params.sort.split(',');
+			sortList.forEach(sort => {
+				const [sortProp, sortOrder] = sort.split(':');
+				query.orderBy(sortProp, sortOrder);
+				query.whereNotNull(sortProp);
+			});
+		}
+
+		if (params.distinct) {
+			const { column } = params.distinct;
```

Comment:
Also, we can probably directly assign the columns to `params.distinct` (similar to `params.sort`) as a CSV instead of a nested object.